lookout/core/tests/test_lib.py

```diff
@@ -124,6 +124,7 @@ def debug(self, *args, **kwargs):
 
     def test_filter_files(self):
         files = [File(path="one", content=b"hello"), File(path="two", content=b"world" * 100)]
+        files = {file.path: file for file in files}
```

Comment:
Can you please add a test which checks that the sequence of 1000 filtered files remains the same. That is,

```python
def create_files():
    files = [File(path="one", content=b"hello"), File(path="two", content=b"world" * 100)] * 1000
    files = random.sample(files, k=len(files))  # note: no need to set the seed
    return {file.path: file for file in files}

files1 = filter_files(create_files())
files2 = filter_files(create_files())
self.assertEqual(files1, files2)
```